tests/vereya/test_observation.py

```diff
@@ -76,8 +76,9 @@ def test_observation_from_chat(self):
         self.assertEqual(command[0], "get wooden_axe")
 
     def test_observation_from_item(self):
-        item_list = self.rob.mc.getItemList()
+        item_list, recipes = self.rob.mc.getItemAndRecipeList()
```

Comment:
does it work without calling observeProc?

Reply:
@noskill  fixed. Now works